examples/charts/file/steps.py

```diff
@@ -1,20 +1,18 @@
-from collections import OrderedDict
+from bokeh.charts import Step, show, output_file
 
-from bokeh._legacy_charts import Step, show, output_file
+# build a dataset where multiple columns measure the same thing
+data = dict(python=[2, 3, 7, 5, 26, 221, 44, 233, 254, 265, 266, 267, 120, 111],
+            pypy=[12, 33, 47, 15, 126, 121, 144, 233, 254, 225, 226, 267, 110, 130],
+            jython=[22, 43, 10, 25, 26, 101, 114, 203, 194, 215, 201, 227, 139, 160],
+            test=['foo', 'bar', 'foo', 'bar', 'foo', 'bar', 'foo', 'bar', 'foo', 'bar',
+                  'foo', 'bar', 'foo', 'bar']
+            )
 
-xyvalues = OrderedDict(
-    python=[2, 3, 7, 5, 26, 81, 44, 93, 94, 105, 66, 67, 90, 83],
-    pypy=[12, 20, 47, 15, 126, 121, 144, 333, 354, 225, 276, 287, 270, 230],
-    jython=[22, 43, 70, 75, 76, 101, 114, 123, 194, 215, 201, 227, 139, 160],
-)
+# create a line chart where each column of measures receives a unique color and dash style
+line = Step(data, y=['python', 'pypy', 'jython'],
+            dash=['python', 'pypy', 'jython'],
+            color=['python', 'pypy', 'jython'],
+            title="Interpreter Sample Data", ylabel='Duration', legend=True)
 
-# any of the following commented are also valid Step inputs
-#xyvalues = pd.DataFrame(xyvalues)
-#xyvalues = list(xyvalues.values())
-#xyvalues = np.array(list(xyvalues.values()))
-
-output_file("steps.html", title="line.py example")
-
-chart = Step(xyvalues, title="Steps", ylabel='measures', legend='top_left')
-
-show(chart)
+output_file("steps.html")
+show(line)
```

Comment:
TBH I find this example extremely weird because the synthetic data is really weird. Is there some better example we can construct? Or maybe just any kind of more meaningful title would help.